helmfile/50-applications.yaml

```diff
@@ -515,7 +515,7 @@ releases:
     app: thanos-ruler
   chart: ./charts/thanos/ruler
   version: 0.1.0
-  installed: {{ and .Values.thanos.enabled .Values.thanos.ruler.enabled }}
+  installed: {{ .Values.thanos.ruler.enabled }}
```

Comment:
Should the ruler not be disabled if you disable thanos?

Reply:
With #849 the option to disable Thanos will be gone.